src/backend/empty/src/lib.rs

```diff
@@ -427,19 +485,20 @@ impl device::Device<Backend> for Device {
     }
 
     unsafe fn map_memory(&self, _: &(), _: memory::Segment) -> Result<*mut u8, device::MapError> {
-        panic!(DO_NOT_USE_MESSAGE)
+        // 640 KB ought to be enough
```

Comment:
that's awesome! would be good to have some sort of safe guards in place for this to not be overrun, still